docs/configure.md

```diff
@@ -236,3 +236,26 @@ region_str = "" if region_index == 0 else sep + short_region_name
 print(account_name + sep + role_name + region_str)
 ```
 If this was stored as `profile_formatter.py`, it could be used as `--profile-name-process "python profile_formatter.py"`
+
+### Change the case of profile names
+
+If you don't want to run (and maintain) a process completely outside of the
+`--profile-name-process`, an option between trimming account & role names and
+executing a separate process is available with `--account-name-case` and
+`--role-name-case`. Use these to change the generated profile names after
+assembly and trimming.
+
+Case modifications include:
```

Comment:
"The values for these options are as follows, and correspond to the Python string methods of the same names:"
And then link each one to the string method docs